standards/webhooks.md

```diff
@@ -0,0 +1,48 @@
+# Webhooks
+
+## Overview
+
+Webhooks are automated messages sent from apps when something happens, typically used to integrate two disconnected systems together, such as with a third-party provider. They have a message—or payload—and are sent to a unique URL configured by the customer or user. Webhooks are an advantage in an API as they are almost always faster than polling for an updated status or parsing to determine which events or changes are new within a timeline. 
+
+- Services **SHOULD** implement push notifications, HTTP callbacks, and other event notifications via webhooks.
+- Services **MAY** implement HTTP callback endpoints as a webhook consumer from other systems and/or APIs.
+
+```note
+Push notification via HTTP Callbacks, often called Web Hooks, to publicly-addressable servers.
+```
+
+## Webhook Consumer
+
+Webhook consumption is necessary to integrate with other first-party and third-party systems. Often building specific API endpoints is necessary to receive webhook-specific payloads, headers and authorization in which the API producer cannot control or change. These webhook specific integration endpoints are usually not RESTful resources and may be required to deviate from expected design patterns within this API Style Guide. As a result webhook consumers should recognize this and organize webhook consuming endpoints accordingly.
```

Comment:
"Often building specific API endpoints is necessary to receive webhook-specific payloads, headers and authorization in which the API producer cannot control or change."
>
 "Often building API endpoints to receive webhook HTTP requests requires implementation of request schema, headers and authorization that is different from current API style and standards as webhooks requests are owned by it's producer and not a consumer."... 
 
 Does this version hits the point you trying to make here?

Reply:
Or you have a lot of these details down below, so may be enough to say "webhook endpoint might look and feel outside of style of API where it's implemented"